React/Base/RCTConvert.m

```diff
@@ -850,7 +850,11 @@ + (UIColor *)UIColor:(id)json
     if ((value = [dictionary objectForKey:@"semantic"])) {
       if ([value isKindOfClass:[NSString class]]) {
         NSString *semanticName = value;
-        UIColor *color = RCTColorFromSemanticColorName(semanticName);
+        UIColor *color = [UIColor colorNamed:semanticName];
```

Comment:
This API was officially added for iOS 11, but we still officially support iOS 10+. This should probably be guarded by `if (@available(iOS 11.0, tvOS 11.0, *))`.

Reply:
@danilobuerger Following up to see if you can update this. Thanks!

Reply:
@appden This seems no longer neccessary with https://github.com/facebook/react-native/commit/a1d626739d95d6cbbb1be169b93952cdd1465486 ?